packages/labs/ssr/src/lib/dom-shim.ts

```diff
@@ -141,14 +141,12 @@ export const getWindow = ({
 
     // Set below
     window: undefined as unknown,
-    global: undefined as unknown,
 
     // User-provided globals, like `require`
     ...props,
   };
 
   window.window = window;
-  window.global = window; // Required for node-fetch
```

Comment:
What about node-fetch here? Should we do something like this?
```ts
Object.assign(window.global, window):
```

Reply:
See PR description -- unless you know a reason it actually fails?

If we do need it, I think we can do `window.global = globalThis`